ansible/roles/ssh/tasks/main.yaml

```diff
@@ -22,5 +22,14 @@
     - { src: "{{jenkins_public_ssh_key_file_path}}",
         dest: "{{jenkins_home_dir}}/.ssh/authorized_keys",
         owner: jenkins, group: jenkins, mode: "0600" }
+
+- name: Add Github and upload server's host key
```

Comment:
This task has no control over whether the file will in fact contain the GitHub and upload server keys. This is up to the user and should be mentioned in the prompt and in the documentation.
I'd name it: "Add known keys for remote hosts" (using a similar wording as in https://en.wikibooks.org/wiki/OpenSSH/Client_Configuration_Files#.7E.2F.ssh.2Fknown_hosts)